README.md

```diff
@@ -196,9 +195,6 @@ npm i
 npm test
 ```
 
-# Implementing using Fetch
-See our documentation for implementing usingFetch [here.](https://docs.fetchoracle.com/fetch/getting-data/introduction)
```

Comment:
See our documentation for implementing usingFetch [here.](https://docs.fetchoracle.com/fetch/getting-data/introduction)